anyway/views/news_flash/api.py

```diff
@@ -153,3 +204,54 @@ def single_news_flash(news_flash_id: int):
             json.dumps(news_flash_obj.serialize(), default=str), mimetype="application/json"
         )
     return Response(status=404)
+
+
+def get_supported_resolutions() -> set:
+    return set([x.value for x in BE_CONST.SUPPORTED_RESOLUTIONS])
+
+
+def get_news_flash_by_id(id: int, query):
+    news_flash_with_id = query.filter(NewsFlash.id == id).first()
+    if news_flash_with_id is None:
+        return Response(status=404)
+    if not is_news_flash_resolution_supported(news_flash_with_id):
+        return Response("News flash location not supported", 406)
+    return Response(
+        json.dumps(news_flash_with_id.serialize(), default=str), mimetype="application/json"
+    )
+
+
+def filter_news_flash_by_interurban_only(query, interurban_only: bool):
+    return query.filter(NewsFlash.resolution.in_(["כביש בינעירוני"]))
```

Comment:
I suggest, if possible, using Backend const instead of the text "כביש בינעירוני": BE_CONST.ResolutionCategories.SUBURBAN_ROAD.value

Reply:
No problem, I left it as it was in the original file because I wanted to focus on the bug fix 
and leave the clean up and refactor to another PR.
If you prefer I will change it here

Reply:
There are several resolutions that are not urban. You can look at anyway/backend_constants.py:96 for the list of resolutions.